js/mandelbrot.js

```diff
@@ -1,17 +1,17 @@
-const WIDTH = 1200, HEIGHT = 800;
+const WIDTH  = 1200;
```

Comment:
at runtime the JS implementation is rendering a different image (compared to the C / WASM implementation) - looks like the colour scale has changed.

Reply:
Fixed